### DISCUSSION TITLE
Issue with kconfig.py - KeyError: 'WARN_EXPERIMENTAL'

### DISCUSSION THREAD

**Discussion by a-domenick** [zephyrproject-rtos/zephyr#52648] [Q&A]
Hi - 
I'm trying to run some tests on the native_posix board, but when compiling my project I run into an issue with Kconfig where it can't find a symbol, even if I set CONFIG_WARN_EXPERIMENTAL=y in my prj.conf file 

> Traceback (most recent call last):
>   File "/home/xyz-workspace/zephyr/scripts/kconfig/kconfig.py", line 278, in <module>
>     main()
>   File "/home/xyz-workspace/zephyr/scripts/kconfig/kconfig.py", line 66, in main
>     if kconf.syms['WARN_EXPERIMENTAL'].tri_value == 2:
> KeyError: 'WARN_EXPERIMENTAL'
> CMake Error at /home/xyz-workspace/zephyr/cmake/kconfig.cmake:272 (message):
>   command failed with return code: 1

Any ideas if I'm missing anything?

**Comment by mkettn** [comment]
You probably created a Kconfig in your app and forgot to add:

```
menu "Zephyr Kernel"
  source "Kconfig.zephyr"
endmenu

```
